chrome/content/otr.js

```diff
@@ -276,6 +284,18 @@ var otr = {
     return human.readString();
   },
 
+  base64encode: function(data, dataLen) {
+    let buf = ctypes.char.array(Math.floor(((dataLen+2)/3)*4 + 1))();
+    let size = libOTR.otrl_base64_encode(buf, data, dataLen);
+    return buf.readString();
```

Comment:
Yes, it seems like we can assume that.  Looks like it in the source,
https://github.com/mozilla/gecko-dev/blob/master/js/src/ctypes/CTypes.cpp#L7616-L7620

But it would probably be best to leave a comment as such.